src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_data_containers.h

```diff
@@ -435,14 +438,14 @@ namespace sirf {
 		}
 		static void init() 
```

Comment:
This function does nothing any more, right? Is it still necessary?

Reply:
@johannesmayer no it is not, just forgot to remove